src/core/router/identity.h

```diff
@@ -292,9 +292,21 @@ struct XORMetric {
   }
 };
 
+/// @brief Hash identity with appended mod date according to spec
+/// @param ident Identity hash used as the basis for the routing key
+/// @throw std::invalid_argument if supplied a zero-initialized ident
+/// @throw std::ios_base::failure on date formatting failure
+/// @throw CryptoPP::Exception on hashing failure
 IdentHash CreateRoutingKey(
     const IdentHash& ident);
 
+/// @brief Format routing key with mod date according to spec
+/// @param ident Identity hash used as the basis for the routing key
+/// @throw std::invalid_argument if supplied a zero-initialized ident
+/// @throw std::ios_base::failure on date formatting failure
+std::string FormatRoutingKey(
+    const IdentHash& ident);
```

Comment:
Is this really greater than 80 chars (it's 53, clang-format shouldn't do this)?

Reply:
No, I'm not sure why the line broke here, I'll fix it.